Revise: improve Variables icon

SlySven opened the pull request:
This adds a white outline around the otherwise entirely black icon so that it becomes visible when used with a dark desktop theme.

For the record this icon was created from scratch on a 256x256 transparent background using The G.I.M.P. to draw "X=" in text in black using the "Serif" font at a size of 150, that image was then redrawn in white four times and each of them was offset by +/-4 in complimentary X and Y directions so that they expanded the background white around the black foreground.

The reason for expanding from the original 48x48 pixel size was to make the anti-aliasing take up a much smaller proportion of the image - so it will appear sharper even at higher resolutions.

This revision came about as a result of a comment made by @Faenriis (here) / **@ Githzerai#4990** (Discord) at 2020/06/12 22:47 in the **# mudlet-development** channel about it not being visible with a dark desktop environment.

This is the original:
![Existing variables icon on light kvantum DE](https://user-images.githubusercontent.com/6163092/84560187-4de7cf00-ad39-11ea-94db-0068ce669988.png) ![Existing variables icon on dark kvantum DE](https://user-images.githubusercontent.com/6163092/84560189-4e806580-ad39-11ea-94bc-f3199d1d4e81.png)

And the proposed replacement:
![Newer variables icon on light kvantum DE](https://user-images.githubusercontent.com/6163092/84560195-5d671800-ad39-11ea-9559-fcafce69acef.png) ![Newer variables icon on dark kvantum DE](https://user-images.githubusercontent.com/6163092/84560197-5dffae80-ad39-11ea-964b-3410deeab5ba.png)

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Revise: improve Variables icon

This adds a white outline around the otherwise entirely black icon so that
it becomes visible when used with a dark desktop theme.

For the record this icon was created from scratch on a 256x256 transparent
background using The G.I.M.P. to draw "X=" in text in black using the
"Serif" font at a size of 150, that image was then redrawn in white four
times and each of them was offset by +/-4 in complimentary X and Y
directions so that they expanded the background white around the black
foreground.

The reason for expanding from the original 48x48 pixel size was to make the
anti-aliasing take up a much smaller proportion of the image - so it will
appear sharper even at higher resolutions.

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

Xabre666 commented:
Thanks for the work. Waiting on test builds to give it an actual look, but it does look a lot better on screenshots you provided.
On that note, **variable.png**  and **function.png** might be given the same treatment because they can suffer from the same problem when using a dark colour scheme, and become notably less visible in the sidebar while using the variables editor.

SlySven commented:
I'll see if I can do over those other two in the same way...

SlySven committed:
Revise: improve variable icon

Does the same thing to the icon used for a variable (in the Variable view
in the editor)

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Revise: improve function icon

Does the same thing to the icon used for a function (in the Variable in the
editor)

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Revise: improve table icon

Does the same thing to the icon used for a table (in the Variable in the
editor)

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Merge branch 'development' into Revise_improveVariablesIcon

Needed to import fix for CI build failures on Travis MacOs

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

Xabre666 commented:
Waiting on test buids seems will take while, so unlazied myself and compiled it on my own from your branch. Just to confirm: new, revised icons work far better, afaic this resolves the issue.

Kebap commented:
There was a brief outage in the test build production which has been fixed already, but requires another commit here to start building anew. Thanks for compiling & testing yourself 👍

keneanung commented:
/refresh links
